public/js/controllers/charts_controller.js

```diff
@@ -208,6 +218,8 @@ function zipXYZData (gData, isHeightAxis, isDayBinned, yCoefficient, zCoefficien
 
 function percentStakedFunc (gData, isHeightAxis, isDayBinned) {
   var extremeStaked = 0
+  rawPoolValue = []
+  rawCoinSupply = []
```

Comment:
Could you reset these before every new chart is loaded? Somewhere around https://github.com/dmigwi/dcrdata/blob/17fdf489f8c49fb899772da364eb88bc9fbe4bd1/public/js/controllers/charts_controller.js#L412